src/routes/buildManifestObject.js

```diff
@@ -12,10 +12,10 @@ import {
 
 const INITIAL_RESULT_COUNT = 100;
 
-function buildQuery(queryParams, pathComponents, resourceData, logger, props) {
+export function buildQuery(queryParams, pathComponents, resourceData, logger, props) {
   const { indexes, sortMap, filters } = getFilterConfig(queryParams.segment);
   const query = { ...resourceData.query };
-  const queryIndex = get(queryParams, 'qindex', 'all');
+  const queryIndex = get(queryParams, 'qindex', 'all') || 'all';
```

Comment:
@YevheniiMaltsev the previous like should already return `all` if `queryParams.qindex` is not present. Perhaps we can refactor it to:

`queryParams?.qindex ?? 'all'`